Fix wrong default values for cron_consumers_runner (#5566)
magento/devdocs: guides/v2.3/config-guide/mq/manage-message-queues.md

#### Behavior by default

> * Cron job `consumers_runner` is disabled

Reviewer: ```suggestion
* Cron job `consumers_runner` is enabled
```

...
```

> * `cron_run` - the option for enabling/disabling cron job `consumers_runner`, by default is false.

Reviewer: ```suggestion
*  `cron_run` - A boolean value that enables or disables the consumers_runner cron job (default = `false`).
```

> * `max_messages` - the maximum number of messages for each consumer that must be processed before consumer terminate, by default is 1000. If it is 0, then the consumer never stops working.

Reviewer: ```suggestion
* `max_messages` - A number specifying the maximum number of messages each consumer must process before terminating (default = `1000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.
```

> * `consumers` - the list of consumers which will be run, by default is empty array (all consumers are allowed to be run).

Reviewer: ```suggestion
* `consumers` - An array of strings specifying which consumer(s) to run. An empty array runs all consumers.
```

Reviewer: @brunoroeder  Thanks for your contribution. I added some suggested updates to clarify the option descriptions, which match the descriptions that are in the _Cloud Guide_ environment variable information related to the `cron_consumer_runner` job.  See [CRON_CONSUMERS_RUNNER](https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html#cron_consumers_runner).

Author: Thanks @meker12

Reviewer: @brunoroeder One small suggestion to add code syntax to `consumers_runner` job.  After you accept those changes, we can merge your PR.  Thx!

...
```

> * `cron_run` - A boolean value that enables or disables the consumers_runner cron job (default = `false`).

Reviewer: ```suggestion
* `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `false`).
```

> * `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `false`).

Reviewer: ```suggestion
* `cron_run` - A boolean value that enables or disables the `consumers_runner` cron job (default = `true`).
```

> * `max_messages` - A number specifying the maximum number of messages each consumer must process before terminating (default = `1000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.

Reviewer: ```suggestion
* `max_messages` - The maximum number of messages each consumer must process before terminating (default = `10000`). Although we do not recommend it, you can use 0 to prevent the consumer from terminating.
```
